lib/fastlane/plugin/wpmreleasetoolkit/helper/git_helper.rb

```diff
@@ -64,6 +64,16 @@ def self.commit(message:, files: nil, push: false)
         Action.sh("git", "commit", *args, "-m", message)
         Action.sh("git", "push", "origin", "HEAD") if push
       end
+
+      # Creates a tag for the given version, and optionally push it to the remote.
+      #
+      # @param [String] version The name of the tag to push, e.g. "1.2"
+      # @param [Bool] push If true 9the default), the tag will also be pushed to `origin`
```

Comment:
`s/9the default)/(the default)` (GitHub isn't showing me the suggestions UI for some reason, sorry)

Reply:
```suggestion
      # @param [Bool] push If true (the default), the tag will also be pushed to `origin`
```